### PR TITLE
Fix SVG icons

### PR DESCRIPTION
**Pull request by Misterio77**
- use interpolations on css custom props
- make svg icons includes instead of imgs
image

### CODE REVIEW

File: _sass/_variables.scss [hunk 1]
```diff
@@ -4,8 +4,8 @@ $black: #0f0f0f;
...
 :root {
     //I would like to use sass variables istead of hardcoding the colors
-    --color-github-icon: white;
-    --color-linkedin-icon: white;
-    --color-telegram-icon: white;
-    --color-twitter-icon: white;
+    --color-github-icon: #{$white};
+    --color-linkedin-icon: #{$white};
+    --color-telegram-icon: #{$white};
+    --color-twitter-icon: #{$white};
```
Comment (Misterio77) [inline _sass/_variables.scss:10]: https://sass-lang.com/documentation/style-rules/declarations#custom-properties

File: _includes/icons/github.svg [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 24 24">
```
Comment (Misterio77) [inline _includes/icons/github.svg:1]: This lets the image size be dynamic

File: _includes/footer.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
             <span>
                 {% for item in site.data.socialMedia %}
                 <a href="{{ item.link }}" class="social-item">
-                    <img src="{{ item.source | relative_url }}" alt="{{ item.name | capitalize }} Logo">
+                    {% include {{ item.source }} %}
```
Comment (Misterio77) [inline _includes/footer.html:10]: You can "include" svgs like regular partials, this will make the SVGs inline, allowing us to style them and much more.

**Comment by Ryrden** [comment]
You're amazing bro, thanks a lot

File: _includes/footer.html (hunk 3 above)
Comment (Ryrden) [reply _includes/footer.html:10]: Cool, but I would like to keep the alt text capitalized anyway, thanks a lot

**Review by Ryrden — APPROVED** [review]
LGTM

File: _includes/icons/github.svg (hunk 2 above)
Comment (Ryrden) [reply _includes/icons/github.svg:1]: You're right, it was a mistake

**Pull request merged**